src/index.js

```diff
@@ -1 +1,15 @@
 import './style.css';
+import addScore from './modules/submitScore.js';
+import displayScores from './modules/displayScore.js';
+
+// const submitBtn = document.querySelector('.submit-btn');
```

Comment:
- Please remove all commented code from your project, this would help ensure that your code is clean and easy to maintain. Please fix this 👍 👍